test/cleanup

```diff
@@ -2,4 +2,15 @@
 
 source $(dirname $0)/common
 
-ursula "envs/test" ${ROOT}/playbooks/${TEST_ENV}/tasks/delete.yml
+stack_exist=`heat stack-list |grep " ${testenv_heat_stack_name} " | wc -l`
```

Comment:
We should switch these `heat` calls into `openstack stack` calls, and stop relying on the `heat` CLI tool.